toolchain/check/scope_stack.h

```diff
@@ -111,9 +111,10 @@ class ScopeStack {
   // is already a `returned var`, returns it instead.
   auto SetReturnedVarOrGetExisting(SemIR::InstId inst_id) -> SemIR::InstId;
 
-  // Looks up the name `name_id` in the current scope. Returns the existing
-  // lookup result, if any.
-  auto LookupInCurrentScope(SemIR::NameId name_id) -> SemIR::InstId;
+  // Looks up the name `name_id` in the current scope and enclosing scopes, but
+  // do not look past `scope_index`. Returns the existing lookup result, if any.
```

Comment:
```suggestion
  // do not look in or past `scope_index`. Returns the existing lookup result, if any.
```
I think this is what the code says?

Reply:
It fails (returns `Invalid`) if the innermost result's scope index is less than (that is, if the scope encloses) `scope_index`, and succeeds if the result's scope is `>= scope_index`, so I think this comment is correct.

Reply:
Oh yes, thanks, I read that backward.